uitools/cpp/Esri/ArcGISRuntime/Toolkit/UtilityNetworkTraceController.cpp

```diff
@@ -758,13 +666,18 @@ void UtilityNetworkTraceController::resetTraceResults()
 {
   setIsResetResultsEnabled(false);
 
-  /*** CLEARING GEOMETRY TRACE RESULTS ***/
+  // Clearing GEOMETRY TRACE RESULTS
+  auto graphics = m_resultsGraphicsOverlay->graphics();
+  for (int i = 0; i < graphics->size(); ++i)
+  {
+    delete graphics->at(i);
```

Comment:
I think you created a specific parent for these graphics - you could potentially just delete that instead to clean these up. This is fine though

Reply:
Not for these graphics, alas - their parent is `this`